src/EmailCampaigns/Newsletter/Newsletter.php

```diff
@@ -14,6 +14,7 @@ public function __construct()
         parent::__construct();
 
         add_action('admin_init', [$this, 'newsletter_sending_handler']);
+        add_action('init', [$this, 'register_post_meta']);
```

Comment:
@picocodes I gave a solution here so we have all code related to this feature in one place. https://github.com/mailoptin/core/commit/2e24b1dd56433f28ed1a30a938d207d282f1b5c3#commitcomment-34351989

Reply:
@collizo4sky fixed